tests/helper.py

```diff
@@ -73,19 +79,19 @@ def dotproduct_adjointness_test(
     torch.testing.assert_close(dotproduct_range, dotproduct_domain, rtol=relative_tolerance, atol=absolute_tolerance)
 
 
-def operator_isometry_test(
-    operator: Operator, u: torch.Tensor, relative_tolerance: float = 1e-3, absolute_tolerance=1e-5
+def linear_operator_isometry_test(
+    linear_operator: LinearOperator, u: torch.Tensor, relative_tolerance: float = 1e-3, absolute_tolerance=1e-5
```

Comment:
why is this limited to linear_operator?
this tests should work for  any operators first output